src/i18n/ar.json

```diff
@@ -0,0 +1,195 @@
+    "Hyogo": "هيوجو",
+    "Ibaraki": "إباراكي",
+    "Ishikawa": "إشوكاوا",
+    "Iwate": "إواتى",
+    "Kagawa": "كاجاوا",
+    "Kagoshima": "كاجوشيما",
+    "Kanagawa": "كاناجاوا",
+    "Kochi": "كوتشي",
+    "Kumamoto": "كوماموتو",
+    "Kyoto": "كيوتو",
+    "Mie": "ميا",
+    "Miyagi": "مياجي",
+    "Miyazaki": "ميازاكي",
+    "Nagano": "ناجانو",
+    "Nagasaki": "ناجازاكي",
+    "Nara": "نارا",
+    "Niigata": "نيجاتا",
+    "Oita": "أوتا",
+    "Okayama": "أوكاياما",
+    "Okinawa": "أوكيناوا",
+    "Osaka": "أوساكا",
+    "Saga": "ساجا",
+    "Saitama": "سايتاما",
+    "Shiga": "شيجا",
+    "Shimane": "شيمانى",
+    "Shizuoka": "شيزووكا",
+    "Tochigi": "طوتشيجي",
+    "Tokushima": "طوكوشيما",
+    "Tokyo": "طوكيو",
+    "Tottori": "طوتوري",
+    "Toyama": "طوياما",
+    "Wakayama": "واكاياما",
+    "Yamagata": "ياماجاتا",
+    "Yamaguchi": "ياماجوتشي",
+    "Yamanashi": "ياماناشي"
+  },
+  "countries": {
+    "albania": "ألبانيا",
+    "andorra": "أندورا",
+    "antiguaandbarbuda": "أنتيغوا وبربودا",
+    "australia": "أستراليا",
+    "austria": "النمسا",
+    "bahrain": "البحرين",
+    "barbados": "بربادوس",
+    "belarus": "بيلاروس",
+    "belgium": "بلجيكا",
+    "bolivia": "بوليفيا",
+    "bosniaherzegovina": "البوسنة والهرسك",
+    "brazil": "البرازيل",
+    "brunei": "بروناي",
+    "bulgaria": "بلغاريا",
+    "canada": "كندا",
+    "chile": "شيلي",
+    "china": "الصين (بما في ذلك هونغ كونغ وماكاو)",
+    "congo-kinshasa": "جمهورية الكونغو الديموقراطية",
+    "cotedivoire": "ساحل العاج",
+    "croatia": "كرواتيا",
+    "cyprus": "قبرص",
+    "czechrepublic": "جمهورية التشيك",
+    "denmark": "الدنمارك",
+    "djibouti": "جيبوتي",
+    "dominica": "دومينيكا",
+    "dominicanrepublic": "جمهورية الدومنيكان",
+    "ecuador": "إكوادور",
+    "egypt": "مصر",
+    "estonia": "إستونيا",
+    "finland": "فنلندا",
+    "france": "فرنسا",
+    "germany": "ألمانيا",
+    "greece": "اليونان",
+    "hungary": "هنغاريا",
+    "iceland": "أيسلندا",
+    "indonesia": "إندونيسيا",
+    "iran": "إيران",
+    "ireland": "أيرلندا",
+    "israel": "إسرائيل",
+    "italy": "إيطاليا",
+    "korea": "كوريا",
+    "kosovo": "كوسوفو",
+    "kuwait": "الكويت",
+    "latvia": "لاتفيا",
+    "liechtenstein": "ليختنشتاين",
+    "lithuania": "ليتوانيا",
+    "luxembourg": "لوكسمبورج",
+    "malaysia": "ماليزيا",
+    "malta": "مالطا",
+    "mauritius": "موريشيوس",
+    "moldova": "مولدوفا",
+    "monaco": "موناكو",
+    "montenegro": "الجبل الأسود",
+    "morocco": "المغرب",
+    "netherlands": "هولندا",
+    "newzealand": "نيوزيلندا",
+    "northmacedonia": "مقدونيا الشمالية",
+    "norway": "النرويج",
+    "oman": "سلطنة عمان",
+    "panama": "بنما",
+    "peru": "بيرو",
+    "philippines": "الفلبين",
+    "poland": "بولندا",
+    "portugal": "البرتغال",
+    "qatar": "قطر",
+    "romania": "رومانيا",
+    "russia": "روسيا",
+    "saintkittsandnevis": "سانت كيتس ونيفيس",
+    "sanmarino": "سان مارينو",
+    "saudiarabia": "Saudi Arabia",
```

Comment:
Is there a reason why this one is not translated?

Reply:
I am sorry for that, I have corrected it